ui/desktop/app/helpers/time-remaining.js

```diff
@@ -0,0 +1,44 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+export default class extends Helper {
+  // =services
+  @service clockTick;
+  @service intl;
```

Comment:
@calcaide @cameronperera do we need to use translations the strings hours, minutes, and seconds?

Reply:
Translations have been added using the `intl` service